manifests/t/TeamLidarr/Lidarr/2.8.2.4493/TeamLidarr.Lidarr.installer.yaml

```
@@ -0,0 +1,31 @@
+# Created using wingetcreate 1.9.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: TeamLidarr.Lidarr
+PackageVersion: 2.8.2.4493
+InstallerLocale: en-US
+Platform:
+- Windows.Desktop
+InstallerType: inno
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+InstallerSwitches:
+  Custom: /NORESTART
+UpgradeBehavior: install
+AppsAndFeaturesEntries:
+- DisplayName: Lidarr version 2.6.4
+  Publisher: Team Lidarr
+  DisplayVersion: 2.6.4

```

```suggestion
- DisplayName: Lidarr version 2.8.2
  Publisher: Team Lidarr
  DisplayVersion: 2.8.2
```
Purely guessing here.